core/src/edu/asu/c3simulator/util/PlayerImp.java

```diff
@@ -0,0 +1,89 @@
+/**
+ * 
+ */
+package edu.asu.c3simulator.util;
+
+import edu.asu.c3simulator.simulation.C3Simulation;
+import edu.asu.c3simulator.simulation.Player;
+import edu.asu.c3simulator.simulation.SocioeconomicStatus;
+
+/**
+ * Implementation of the Player interface
+ * 
+ * @author Reigel, Justin
+ *
+ */
+
+public class PlayerImp implements Player
```

Comment:
Naming these classes is always difficult. We should think of a better name than PlayerImp, though.